contrib/src/main/java/com/datatorrent/contrib/hbase/HBaseWindowStore.java

```diff
@@ -132,8 +140,11 @@ public void beginTransaction() {
 
   @Override
   public void commitTransaction() {
-    // HBase does support transactions so this method left empty
-
+    try {
+      flushTables();
```

Comment:
Is it correct to just flushTables for commitTransaction when we don't have any other transaction semantics implemented? e.g. beginTransactions, rollback etc are empty and do nothing. So implementing commitTransaction() like this is misleading?